ArcadiaParties-SPA/src/app/content/content.component.scss

```diff
@@ -0,0 +1,5 @@
+:host {
+  overflow: auto;
```

Comment:
This is not content component's duty to set overflow. It is content container's responsibility.

Reply:
I've fixed this problem.